source/_integrations/dlna_dmr.markdown

```diff
@@ -14,9 +18,24 @@ The `dlna_dmr` platform allows you to control a [DLNA Digital Media Renderer](ht
 
 Please note that some devices, such as Samsung TVs, are rather picky about the source used to play from. The TTS service might not work in combination with these devices. If the play_media service does not work, please try playing from a DLNA/DMS (such as [MiniDLNA](https://sourceforge.net/projects/minidlna/)).
 
-## Configuration
+{% include integrations/config_flow.md %}
 
-To add a DLNA DMR device to your installation, add the following to your `configuration.yaml` file:
+## Options
+
+Options for DLNA DMR devices can be set going to **Configuration** -> **Integrations** -> **DLNA Digital Media Renderer** -> **Configuration**.
+
+{% configuration_basic %}
+Event listener port:
+  description: "Local port to listen on for events sent by the DLNA device. If this is not set, a random port will be allocated. Use this if you need a specific incoming port for firewall or NAT reasons."
+Event listener callback URL:
+  description: "Local URL destination for events sent by the DLNA device. It should be of the form `http://{host}:{port}/notify`, where keywords `{host}` and `{port}` will be automatically filled-in but can be set explicitly here, e.g. `http://192.88.99.1:5555/notify`. Use this if the local IP address or port seen by Home Assistant is not what the device should connect to, because of Network Address Translation (NAT)."
+Poll for device availability:
+  description: "Periodically try to connect to the DLNA device, even if it is unavailable. Enable this if SSDP advertisements sent by the device are not received by Home Assistant, e.g. when IP multicast is broken on your network."
+{% endconfiguration_basic %}
+
+## Manual Configuration
+
+Alternatively, you can add a DLNA DMR device to your installation, by adding the following to your `configuration.yaml` file:
```

Comment:
When an integration gets configuration via the UI, the YAML configuration must be deprecated. As we don't keep deprecated documentation around, it means this part should be fully removed.

Reply:
That makes sense. I've removed the old configuration method, as you suggested.

Reply:
Should the actual implementation in the component supporting this be removed as well?

Reply:
Judging from [other integrations](https://www.home-assistant.io/blog/2021/06/02/release-20216/#integrations-that-deprecated-yaml-import), there is a 2 version grace period for migration of yaml to config entries. I've got that migration logic implemented for now, and it can be removed in a few months time.